docs/application/dotnet/guides/multimedia/media-controller.md

```diff
@@ -7,17 +7,29 @@ The main media controller features include:
 
 -   Updating and retrieving information
 
-    You can [update the metadata and playback information](#get_media) on the server side, and then retrieve that information on the client side.
+    You can [update the metadata and playback information](#updating-and-retrieving-information) on the server side, and then retrieve that information on the client side.
 
     The media control server provides current information about the registered application that you can send to the client.
 
     When the client requests the information, the media control server updates the state information of an active application before transferring the data. If the application is not running when the client request arrives, the media control server transfers the latest information.
 
+- Updating and Retrieving Playlist
+
+    You can [create a playlist](#updating-and-retrieving-playlist) on the server side, and then retrieve that information on the client side.
+
 - Sending and processing commands
 
-    You can [send a command](#send_media) to the server from the client side, and then process the command on the server side.
+    You can [send a command](#sending-and-processing-commands) to the server from the client side, and then process the command on the server side.
+
+    The client can request server state or [metadata](#media-control-metadata-properties) information from the server, and receive it through an event handler.
+
+- Setting and Getting Playback Capability
+
+    You can [set the playback capability](#setting-and-getting-playback-capability) on the server side, and then get that capability on the client side.
 
-    The client can request server state or [metadata](#servermetadata) information from the server, and receive it through an event handler.
+ - Media Control Metadata Properties
```

Comment:
Just a question. I can see we already have a link to "metadata" section above in line 24. Most of this sentence is duplicated with it. So I think adding this line 30 and line 32 doesn't have big difference, as this kind of mis-order between the list and section is acceptable.
What do you think about it?

Reply:
I agree it's duplicated. I'll remove and change statement of `Sending and processing commands`